lib/display.go

```diff
@@ -207,6 +207,11 @@ func highlightAlgorithm(sigAlg simpleSigAlg) string {
 	return color.SprintFunc()(algString(sig))
 }
 
+// timeString formats a time in UTC with minute precision, in the given color.
+func timeString(t time.Time, c *color.Color) string {
+	return c.SprintfFunc()(t.Format("2006-01-02 15:04Z"))
```

Comment:
I would show the date as-is in the cert, i.e. use `MST` in the format string. That'll print UTC for all practical purposes, unless someone explicitly put non-UTC time in a cert.